### ISSUE TITLE
Included Model Types

### ISSUE THREAD

**Issue by jkennedyvz** [autogluon/autogluon#1458]
Hi, 

I would like a feature to specify included_model_types on .fit(). 

I am currently using a very long excluded_model_types list like:
`excluded_model_types = ['KNN', 'NN', 'XT', 'RF', 'FASTAI']`

**Comment by willsmithorg** [comment]
Try this
```
from autogluon.tabular.configs.hyperparameter_configs import get_hyperparameter_config

custom_hyperparameters = get_hyperparameter_config('default')
```
Now delete observe that every model type has a key in the dict.  You can remove those you don't want, or create a simpler hyperparameter dict containing only the models you want.

Then do:
```
from autogluon.tabular import TabularPredictor
predictor = TabularPredictor(label=label).fit(train_data, hyperparameters=custom_hyperparameters)
```

**Comment by willsmithorg** [comment]
Note for developers : users seem to want an included_model_types parameter to make this functionality more intuitive.

**Comment by yinweisu** [comment]
This feature has been added in the mainline and will be available after 0.8 release. An example usage would be 
`predictor = TabularPredictor(label=label).fit(train_data, included_model_types=["KNN"])`

**Comment by jkennedyvz** [comment]
TYSM @yinweisu and team!
